tests/src/test/scala/org/http4s/EntityDecoderSpec.scala

```diff
@@ -188,7 +202,7 @@ class EntityDecoderSpec extends Http4sSpec with PendingUntilFixed {
         decoder1
           .orElse(decoder2)
           .orElse(failDecoder)
-          .decode(Request(), strict = true) must returnLeft(MediaTypeMissing(expectedMediaRanges))
+          .decode(Request[IO](), strict = true) must returnLeft(MediaTypeMissing(expectedMediaRanges))
```

Comment:
We had these requests sitting around that were inferred to nothing. `Request[Nothing](...)` cannot infer the implicit functions if these circumstances